gifski-api/gifski.h

```diff
@@ -53,6 +53,10 @@ typedef struct GifskiSettings {
    * Lower quality, but faster encode.
    */
   bool fast;
+  /**
+   * 0+, but useful range is 1-10. Recommended to set to 5. If times_shown is > 0 once should be set to true
+   */
+  uint16_t times_shown;
```

Comment:
You need to upstream these changes (any changes in the `gifski-api` directory) to https://github.com/ImageOptim/gifski, which is the source of truth.

Reply:
I will add in a pull request for that project as well.